### PR TITLE
Show API scan progress

### LINKED ISSUE #349: [Bug] Scan status is not working properly
image

### LINKED ISSUE #383: API scanning dialogue box doesn't appear the next time

### LINKED ISSUE #384: Add API Scan Progress Indicator

### PR DESCRIPTION
**Pull request by yashwin**
fixes #349 
fixes #383 
fixes #384

### CODE REVIEW

File: app/models/file.coffee [hunk 1]
```diff
@@ -27,12 +27,21 @@ File = DS.Model.extend BaseModelMixin,
...
     !manual
   ).property 'manual'
 
+  isApiScanning: (->
+    apiScanProgress = @get "apiScanProgress"
+    if apiScanProgress in [0, undefined, 100]
+      false
+    else
+      true
+  ).property "apiScanProgress"
```
Comment (dhilipsiva) [inline app/models/file.coffee]: Have a small feedback here. Will share tomorrow.

**Comment by viren-nadkarni** [comment]
@yashwin static scan progress will now be recorded; you can get it in `static-scan-progress` file field.
Also, the fields `is-static-done`, `is-dynamic-done`, `is-api-done` will now be properly set, so you won't have to calculate it.

**Comment by yashwin** [comment]
Awesome @viren-nadkarni

File: app/styles/app.sass [hunk 2]
```diff
@@ -370,7 +371,7 @@ button, .button
         padding: 2px
         border: 1px solid #cccccc!important
         vertical-align: middle
-  :last-child
+  :nth-child(3)
```
Comment (dhilipsiva) [inline app/styles/app.sass]: Use class - because this might cause errors in the future.

**Comment by dhilipsiva** [comment]
@subho007 Any UI feedbacks?

File: app/styles/app.sass (hunk 2 above)
Comment (yashwin) [inline app/styles/app.sass]: @dhilipsiva :https://github.com/appknox/irene/pull/397/commits/dd59c6643b65d8ad29b8e6afcb1500c9451f37d6

**Comment by subho007** [comment]
if the UI feedback is ok will we merge ? @dhilipsiva

**Comment by dhilipsiva** [comment]
@subho007 yeah subho. Merge and test it on staging. Why?

**Comment by dhilipsiva** [comment]
Assuming there are no changes from your end.

**Comment by subho007** [comment]
@dhilipsiva which means after I check I will be merging this branch to develop 👍

**Comment by dhilipsiva** [comment]
No @subho007. I was just expecting you to Approve. I will merge this. Or are you saying that you need to merge this?

**Comment by subho007** [comment]
@dhilipsiva if things look good from your side can you approve ?

**Comment by subho007** [comment]
why did you merge ?:O

**Comment by subho007** [comment]
what is wrong with you @dhilipsiva ?

**Comment by dhilipsiva** [comment]
Huh??? Nothing is wrong with me. I was about to ask the same question. I thought you were on phone and wanted me to do this myself. I already reaquested all changes from my side and the changes have been made.

**Pull request merged**